Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -0,0 +1,402 @@
+﻿# Private module-scope variables.
+$script:azureUri = "https://management.core.windows.net"
+$script:azureRmUri = "https://management.azure.com"
```

Comment:
Don't hardcode. Can you use this from the endpoint object. We will break support for other Azure Cloud environment types.